### PR TITLE
docs: update content

### PR DESCRIPTION
**Pull request by fleuraly**
Update [intro] of Logto documentation.

## Checklist

- [ ] `.changset-staged`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [ ] This PR is not applicable for the checklist

### CODE REVIEW

File: docs/docs/recipes/manage-users/you-dont-need-a-user-table.md [hunk 1]
```diff
@@ -1,39 +1,37 @@
...
-# You don't need a user table
+# Common practice: you don't need a user table 
 
 ## Identify user
 
-We recommend using `user.id` to identify a user since we guarantee every user has a unique and non-null `id` property.
+We suggest utilizing the `user.id` property to identify a user, as we ensure each user has a distinct and non-null `id`. Additionally, the `username`, `primary_email`, and `primary_phone` properties are unique but may be empty. Therefore, it is important to handle `null` values appropriately if you intend to use these properties as identifiers for a user.
 
-Also, `username` `primary_email` and `primary_phone` are unique properties. But they may be empty. Remember to handle `null` properly if you want to use these properties to identify a user.
+## Why is there no user table
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/you-dont-need-a-user-table.md:11]: A little bit misleading. Something like, why you don't need a separate user table

File: docs/docs/recipes/manage-users/you-dont-need-a-user-table.md [hunk 2]
```diff
@@ -1,39 +1,37 @@
...
 
-Traditionally, developers will create a user table in a SQL database containing the profile and other user-related information, then create a many-to-one relation in other tables, a `user` column pointing to the user table. For other identity providers, take Google as an example. Create a column named `google_id` to achieve "sign in with Google" linking. Now, for Logto, you may want to create a column `logto_id`, but it is not a good way since Logto is more than a social provider, it is a solution that can take charge of all user-related stuffs.
+Using Logto eliminates the need for a user table.
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/you-dont-need-a-user-table.md:17]: ditto

File: docs/docs/README.mdx [hunk 3]
```diff
@@ -5,32 +5,32 @@ slug: /
...
 
 - An OIDC-based identity service
-- Multi-platform user sign-in/up experience with dark mode and SDKs (Web, iOS, and Android)
-- Sign-in/up with dynamic SMS/Email passcode
-- Out-of-box social sign-in integration (GitHub, Google, WeChat, Alipay, etc.)
-- A web UI to control all above (Admin Console)
-- Extendable multi-language support
+- User-friendly sign-in experience with customizable CSS
+- Various sign-in options, such as social, email, phone number, and username, including passwordless authentication
+- A management API that is ready to use and can serve as your authentication provider
```
[the remark this answers is not in the document]
Comment (fleuraly) [reply docs/docs/README.mdx]: Updated

File: docs/docs/recipes/manage-users/admin-console.md [hunk 4]
```diff
@@ -5,62 +5,51 @@ sidebar_position: 1
...
-Click the "+ Add User" button in the upper right corner.
-In the opening modal, fill out the username and full name, and click the "Add User" button to confirm.
-
-After you add the user, copy the **username** and **initial password** to send to the end-user.
+After the new user has been created, you can copy their **username** and **initial password** to send to them. This will allow them to log in to the system and start using it.
 
 :::caution
-Once the "Add User" modal is closed, you can no longer view the password.
-If you forget to keep it, you can [reset the password](#reset-user-password).
+f you fail to retain the password after closing the "Add User" modal, you will not be able to view it again. If you forget to keep it, you can [reset the password](#reset-user-password).
```
[the remark this answers is not in the document]
Comment (fleuraly) [reply docs/docs/recipes/manage-users/admin-console.md]: Updated

File: docs/docs/recipes/manage-users/you-dont-need-a-user-table.md (hunk 1 above)
Comment (fleuraly) [reply docs/docs/recipes/manage-users/you-dont-need-a-user-table.md]: Updated

File: docs/docs/recipes/manage-users/you-dont-need-a-user-table.md (hunk 2 above)
Comment (fleuraly) [reply docs/docs/recipes/manage-users/you-dont-need-a-user-table.md:17]: Updated

**Review by gao-sun — APPROVED** [review]

**Pull request merged**